app/controllers/api/v2/compute_attributes_controller.rb

```diff
@@ -1,16 +1,42 @@
 module Api
   module V2
     class ComputeAttributesController < V2::BaseController
+      include Api::Version2
       include Foreman::Controller::Parameters::ComputeAttribute
+      include ParameterAttributes
```

Comment:
I do not see why we `include ParameterAttributes`. What am I missing?

Reply:
That's not needed. removed. When I was adding index, I took a look on subnet API controller to find how nested resources and scoped search were working and I added this but it' not needed. Thanks for the pointer.